corehq/apps/locations/fixtures.py

```diff
@@ -120,7 +120,7 @@ def get_xml_nodes(self, fixture_id, restore_user, locations_queryset, data_field
         )
         location_type_attrs = ['{}_id'.format(t) for t in all_types if t is not None]
         attrs_to_index = ['@{}'.format(attr) for attr in location_type_attrs]
-        attrs_to_index.extend(['@id', '@type'])
+        attrs_to_index.extend(['@id', '@type', 'name'])
```

Comment:
What decides if we need '@' with the name or not?

Reply:
These are acccessors into the xml element, so anything with an @ is an attribute of the root element, and things with no @ are child elements

Reply:
Whether it's an xml attribute or a child node.  If you expand the example test file, you'll see that id, type, and the others are attrs, and name is a child node.

Reply:
Much Thanks! 👍